Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM golang:1.13-alpine
+FROM golang:1.14 as builder
 
-RUN apk update
-RUN apk add git
-RUN apk add gcc
-RUN apk add libc-dev
+WORKDIR /
+ADD go.mod ./
+RUN go mod download
+ADD . .
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-extldflags "-static"' -o transfermeit .
```

Comment:
if you change to `-o /transfermeit` you can make the COPY command a lot smaller bellow:
```
COPY  --from=builder /transfermeit /transfermeit
```

Reply:
Also why not just ?
```
RUN build -o /transfermeit .
```

Reply:
From : https://rollout.io/blog/building-minimal-docker-containers-for-go-applications/ 

> Huh? What does that mean? Took me a while to figure it out, but our Go binary is looking for libraries on the operating system it’s running in. We compiled our app, but it still is dynamically linked to the libraries it needs to run (i.e., all the C libraries it binds to). Unfortunately, scratch is empty, so there are no libraries and no loadpath for it to look in. What we have to do is modify our build script to statically compile our app with all libraries built in: